examples/upload-files/README.MD

```diff
@@ -0,0 +1,25 @@
+# Select Files To Upload
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+TestCafe cannot access the browser's `Choose File` dialog to emulate file selection. For this reason, TestCafe uses another way to select files. This example shows how to select files to populate a `type="file"` input.
```

Comment:
> For this reason, TestCafe uses another way to select files.

After this sentence, I would expect some words about this other way (it's intriguing). However, we say nothing...

I think that the following phrasing creates less intrigue:

*TestCafe cannot access the browser's `Choose File` dialog to emulate file selection. This example shows how to add files to a `type="file"` input without browser dialogs.*

Reply:
I agree. Edited